packages/cli/src/commands/models/m365Unacquire.ts

```diff
@@ -1,57 +1,75 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import { CLICommand, err, ok } from "@microsoft/teamsfx-api";
-import { PackageService } from "@microsoft/teamsfx-core";
+import { UninstallInputs, QuestionNames } from "@microsoft/teamsfx-core";
 import { logger } from "../../commonlib/logger";
 import { MissingRequiredOptionError } from "../../error";
 import { commands } from "../../resource";
 import { TelemetryEvent } from "../../telemetry/cliTelemetryEvents";
 import { m365utils, sideloadingServiceEndpoint } from "./m365Sideloading";
+import { getFxCore } from "../../activate";
 
 export const m365UnacquireCommand: CLICommand = {
   name: "uninstall",
   aliases: ["unacquire"],
   description: commands.uninstall.description,
   options: [
     {
-      name: "title-id",
+      name: QuestionNames.UninstallMode,
+      description: commands.uninstall.options["mode"],
+      type: "string",
+    },
+    {
+      name: QuestionNames.TitleId,
       description: commands.uninstall.options["title-id"],
       type: "string",
     },
     {
-      name: "manifest-id",
+      name: QuestionNames.ManifestId,
       description: commands.uninstall.options["manifest-id"],
       type: "string",
     },
+    {
+      name: QuestionNames.Env,
+      description: commands.uninstall.options["env"],
+      type: "string",
+    },
+    {
+      name: QuestionNames.ProjectPath,
```

Comment:
suggest to use global option `ProjectFolderOption`

Reply:
ProjectFolderOption's "required" property is true, but in cli uninstall, the project path is optional.

Reply:
Though `ProjectFolderOption` is required, it has a current folder as default value, this option should be consistant globally for the TTK CLI.

Reply:
Adjusted the naming. Thanks.